.github/workflows/promote_to_draft.yml

```diff
@@ -69,6 +69,7 @@ jobs:
       image: nycplanning/build-base:latest
     env:
       PUBLISHING_BUCKET: ${{ inputs.dev_bucket && format('de-dev-{0}', inputs.dev_bucket) || 'edm-publishing' }}
+      DEV_FLAG: ${{ inputs.dev_bucket && 'true' || 'false' }}
```

Comment:
Long term this doesn't feel the __most__ sustainable way of handling this, ideally we have a single value somewhere setting the environment and we have a script or something handle translating that to buckets, db connection strings (if we had different db envs), etc... but this is what we currently have in other places and I think it's simplest for now

Reply:
maybe we could also add a condition, not necessarily here, to only run a job if one of the following conditions is true? 

(`DEV_FLAG = false` and `dev_bucket` is an empty string)
or 
(`DEV_FLAG = true` and `dev_bucket` is not empty)

Reply:
or maybe I'm tripping and we don't need it?

Reply:
I would lean with no - it's just adding another thing that needs to be manually specified. Right now, the dev bucket is really the only flag we have, and I think it makes sense to set a dev flag this way. And I'd rather put effort into consolidating logic to generate the various variables related to some sort of overall env flag than to try to add logic to validate that they're consistent